doc/containers.md

```diff
@@ -496,6 +500,30 @@ pool            | string    | -                 | no        | The storage pool t
 propagation     | string    | -                 | no        | Controls how a bind-mount is shared between the container and the host. (Can be one of `private`, the default, or `shared`, `slave`, `unbindable`,  `rshared`, `rslave`, `runbindable`,  `rprivate`. Please see the Linux Kernel [shared subtree](https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt) documentation for a full explanation)
 shift           | boolean   | false             | no        | Setup a shifting overlay to translate the source uid/gid to match the container
 
+#### sourcetype: ceph-rbd
+
+Mount from existing ceph RBD device that is externally managed
+
+Device configuration properties:
+
+Key                     | Type      | Default           | Required  | API extension                          | Description
+:--                     | :--       | :--               | :--       | :--                                    | :--
+ceph.user_name          | string    | -                 | yes       |                                        | username of ceph
+ceph.cluster_name       | string    | -                 | yes       |                                        | cluster name of ceph
+
+
+#### sourcetype: ceph-fs
+
+Mount from existing ceph FS device that is externally managed
+
+Device configuration properties:
+
+Key                     | Type      | Default           | Required  | API extension                          | Description
+:--                     | :--       | :--               | :--       | :--                                    | :--
+ceph.user_name          | string    | -                 | yes       |                                        | username of ceph
+ceph.cluster_name       | string    | -                 | yes       |                                        | cluster name of ceph
+
```

Comment:
So those two sourcetype sections are a bit misleading as they suggest that none of the other disk options apply, which isn't actually the case.

I think it'd be better to just add the two options to the list of options in the `disk` table above, properly list their default values, API extension and only have that paragraph above explain the two source types, the syntax for them and that those two options only apply to such disks.